src/data/mv_polynomial/invertible.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2020 Johan Commelin and Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin and Robert Y. Lewis
+-/
+
+import data.mv_polynomial.basic
+import algebra.invertible
+
+/-!
+# Invertible polynomials
+
+This file is a stub containing some basic facts about invertible elements in the ring of polynomials.
+-/
+
+open mv_polynomial
+
+/-- A natural number that is invertible when coerced to `R` is also invertible
+when coerced to any `R`-algebra. -/
+def invertible_algebra_coe_nat (R S : Type*) (p : ℕ)
+  [comm_semiring R] [semiring S] [algebra R S] [invertible (p : R)] :
+  invertible (p : S) :=
+invertible.copy (invertible.map (algebra_map R S : R →* S) p) p
+  (by simp only [ring_hom.map_nat_cast, coe_monoid_hom])
```

Comment:
Maybe this is a good fit for `ring_theory.algebra_tower`, in something like the following form:

```lean
def invertible.algebra_tower (R S T : Type*) (p : R)
  [comm_semiring R] [comm_semiring S] [semiring T] [algebra R S] [algebra S T] [algebra R T]
  [is_scalar_tower R S T] [invertible (algebra_map R S p)] :
  invertible (algebra_map R T p) :=
invertible.copy (invertible.map (algebra_map S T : S →* T) (algebra_map R S p)) (algebra_map R T p)
  (by rw [coe_monoid_hom, is_scalar_tower.algebra_map_apply R S T])

def invertible_algebra_coe_nat (R S : Type*) (p : ℕ)
  [comm_semiring R] [semiring S] [algebra R S] [inv : invertible (p : R)] :
  invertible (p : S) :=
by { haveI : invertible (algebra_map ℕ R p) := inv, exact invertible.algebra_tower ℕ R S p }
```

Reply:
Great idea! Done.